### PR TITLE
Add Send receive data using API to leaderboard project

### PR DESCRIPTION
**Pull request by imranhossainemi**
**Update at leaderboard project files**

- update the index.js file
- update the index.html file

### CODE REVIEW

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @imranhossainemi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that 

- [ ] misses some headings in your README file. Kindly keep all the headings from the [Microverse template](https://github.com/microverseinc/readme-template) you used and provide appropriate information under each of the headings. You can write `Not available yet`  under any heading that is not related to your current project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by amejid — COMMENTED** [review]
Hi @imranhossainemi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where the README file is missing some relevant sections. The `Acknowledgments` section is missing. I suggest you put the time to work on your README file to make your work more professional. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1 +1,77 @@
...
+  displayUser();
+  return user;
+}
+
+refresh.addEventListener('click', (event) => {
+  event.preventDefault();
+  displayUser();
+});
+
+displayUser();
```
Comment (rotimiazeez) [inline src/index.js]: - Please ensure the usage of `arrow functions` instead of the `function` keyword. 😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @imranhossainemi 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

Please feel free to use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend considering them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Owusu-Desmond — APPROVED** [review]
Hi @imranhossainemi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**